ui-v2/app/styles/components/table.scss

```diff
@@ -19,6 +20,13 @@ html.template-service.template-list main th:first-child {
 td.folder {
   @extend %with-folder;
 }
+td .kind-proxy {
+  @extend %type-icon, %with-proxy;
+  text-indent: -9000px !important;
+  width: 24px;
+  margin-top: -8px;
+  transform: scale(0.7);
```

Comment:
The scale here is a little odd. Is this because the SVG is too large? Why not set the width lower?

Reply:
The scale means I can change the scale of the entire icon in one line, otherwise I'd have to change the width/heights of 2 things in 2 selectors, the gray background which is CSS and the icon itself which is pseudo content, plus the size of the border radius. That would probably be about 10 lines of code as opposed to 1. Plus `scale` does what it says on the tin, and just seemed easier. Is there any reason why not to use scale here?

Reply:
I don't think there's a reason not to aside from it being unexpected (with the expected thing being width/height) and maybe a "wtf" down the road if things don't line up and the `transform: scale` gets overlooked.

Not strong arguments and your reasoning is good, just thought I'd ask.